aiida_pseudo/data/pseudo/psp8.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+"""Module for data plugin to represent a pseudo potential in UPF format."""
+from typing import BinaryIO
+
+from .pseudo import PseudoPotentialData
+
+__all__ = ('Psp8Data',)
+
+
+def parse_element(stream: BinaryIO):
+    """Parse the content of the UPF file to determine the element.
+
+    :param stream: a filelike object with the binary content of the file.
+    :return: the symbol of the element following the IUPAC naming standard.
+    """
+    lines = stream.read().decode('utf-8')
+
+    return lines.split()[0].strip()
```

Comment:
Does this line actually return the element symbol? According to the [documentation](https://docs.abinit.org/developers/psp8_info/) the first line of Psp8 is just a title. Shouldn't you be using the atomic number on the second line to determine the element?

Reply:
In the case of psp generated with ONCVP(so all the psp from PseudoDojo) the first element of the first line (so the Title line) is the element symbol.

Reply:
But this is not the general rule for all pseudo files that respect the Psp8 format. In that case, I would really not rely on this. The format seems to guarantee the presence of the atomic number though. It is easy to transform this to the symbol in a reliable and reproducible way. Something like the following should work (not tested):
```python
from aiida.common.constants import elements

lines = stream.read().decode('utf-8')

try:
    atomic_number = int(lines.split()[1].strip())
except (IndexError, ValueError):
    raise ValueError('failed to parse the atomic number.')

try:
    symbol = elements[atomic_number]['symbol']
except KeyError:
    raise ValueError(f'the atomic number {atomic_number} is not supported.')

return symbol
```

Reply:
Ok sure.